cmake/CMakeLists.txt

```diff
@@ -188,6 +188,8 @@ option(onnxruntime_ENABLE_CPUINFO "Enable cpuinfo" ON)
 # ATen fallback support
 option(onnxruntime_ENABLE_ATEN "Enable ATen fallback" OFF)
 
+option(onnxruntime_BUILD_KERNEL_EXPLORER "Build kernel_explorer for microbenchmark a small subset of important kernels" OFF)
```

Comment:
"Build Kernel Explorer for testing and profiling GPU kernels"

(kernels already implies the benchmarking is at micro level)